src/index.css

```diff
@@ -7,8 +7,13 @@
   min-width: 94%;
 }
 
-.toggle-block svg {
+#toggle-icon svg {
```

Comment:
@juliansg022 as a recommendation you should use a CSS class instead of an ID, there will be a lot of toggle icons in the document, and remember that the IDs should be unique. 

Another suggestion is to use a name that does not interfere with the parent project CSS classes ... something like `.toggle-block__icon`  `.toggle-block__selector`

Reply:
Thank you @juanhurtado10, I'll follow the recommendations